tests/unit/test_fact_manager.py

```diff
@@ -54,14 +53,14 @@ def test_fm_fx(request) -> FactManager:
 
     test_data = [f"INSERT INTO {test_table} (name, lang, message, author, editedby, edited) "
                  f"VALUES ('stats', 'en', 'Fuel Rats Statistics: https://t.fuelr.at/stats',"
-                 f" 'CrunchyBaton955', 'CrunchyBaton955)', '{datetime.datetime(1970, 1, 1, 0, 0, 0)}')",
+                 f" 'CrunchyBaton955', 'CrunchyBaton955)', '{pendulum.DateTime(1970, 1, 1, 0, 0, 0)}')",
```

Comment:
(3x) [style] We should be consistent, do we, or do we not append tz=pendulum.tz.UTC?

Reply:
We don't need to, since its implied.